upa/contracts/UpaProofReceiver.sol

```diff
@@ -243,7 +250,7 @@ contract UpaProofReceiver is
         emit VKRegistered(circuitId, vk);
     }
 
-    // See IUpa.sol
+    // See IUpaProofReceiver.sol
```

Comment:
unnecessary comment

Reply:
I think it's fine to keep it.  It shows which of the multiple parent classes declares this method.